scripts/test_data/generate_qualifier_regression_data.py

```diff
@@ -11,6 +11,7 @@
 import spacy
 
 import clinlp  # noqa: F401
+from clinlp.ie import SPANS_KEY
```

Comment:
"ents" as constant because it may change in the future?

Reply:
Yes, if we ever want to change it it's nice to have it in one place only